calibration/model_interface.jl

```diff
@@ -0,0 +1,65 @@
+import ClimaAtmos as CA
+import YAML
+import ClimaComms
+@static pkgversion(ClimaComms) >= v"0.6" && ClimaComms.@import_required_backends
+using ClimaUtilities.ClimaArtifacts
+import ClimaCalibrate:
+    set_up_forward_model,
+    run_forward_model,
+    path_to_ensemble_member,
+    ExperimentConfig
+
+"""
+    set_up_forward_model(member, iteration, experiment_dir::AbstractString)
+    set_up_forward_model(member, iteration, ::ExperimentConfig)
+    set_up_forward_model(member, iteration, config_dict::AbstractDict)
+
+Return an AtmosConfig object for the given member and iteration.
+
+Turns off default diagnostics and sets the TOML parameter file to the member's path.
+This assumes that the  config dictionary has an `output_dir` key.
+"""
+function set_up_forward_model(member, iteration, ::ExperimentConfig)
+    # Assume experiment_dir is project dir
+    experiment_dir = dirname(Base.active_project())
```

Comment:
There is no reason to assume that the experiment_dir is the same as the active project dir. Actually, in many case it is preferable for the two not to be the same (e.g., you are on a cluster with limited quota in $HOME, so you want to run on a scratch space)